src/test/java/com/easypost/WebhookTest.java

```diff
@@ -108,12 +107,15 @@ public void testRetrieve() throws EasyPostException {
     @Test
     public void testAll() throws EasyPostException {
         vcr.setUpTest("all");
+        Map<String, Object> params = new HashMap<>();
+        params.put("url", Fixtures.webhookUrl());
 
-        WebhookCollection webhooks = vcr.client.webhook.all();
+        vcr.client.webhook.create(params);
```

Comment:
issue: when testing the retrieval of all webhooks, we shouldn't also be creating them. Can we remove this line?

Reply:
It would be an empty list of webhooks. We need to create one to ensure the webhook objects deserialize into a list properly

Reply:
I suppose that's true eh... thanks for clarifying